### DISCUSSION TITLE
ERROR: Could not find acceptable solution

### DISCUSSION THREAD

**Discussion by Hanyu-B** [pybamm-team/PyBaMM#3208] [Q&A]
<img width="733" alt="inconsistant state " src="https://github.com/pybamm-team/PyBaMM/assets/74037661/244997d8-a09b-4e01-98c3-e20304783643">

I received this error when I was trying to use a saved battery pybamm file as the starting solution or initiate the model. I am not sure what the true issue is behind this bug, but I tried different parameter values and the bug would be fixed. I can provide an example of my work if necessary.

Also, sometimes I get another bug that is quite similar to this one, and the other one says that not able to find a consistent state instead.

**Comment by brosaplanella** [comment]
This is probably caused by inconsistent parameters, could you please share a minimum working example?
